setup.sh

```diff
@@ -56,31 +54,12 @@ py3-pip-bash-completion
 py3-pip-doc
 "
 
-UTILITY_PROGRAMS="
-curl
-ffmpeg
-htop
-lynx
-mediainfo
-openssh
-p7zip
-rsync
-rtmpdump
-samba-common
-speedtest-cli
-"
-
-PIP_PROGRAMS="
-youtube-dl
-"
-
+UTILITY_PROGRAMS
```

Comment:
```suggestion
UTILITY_PROGRAMS="
curl
ffmpeg
htop
lynx
mediainfo
openssh
p7zip
rsync
rtmpdump
samba-common
speedtest-cli
"

PIP_PROGRAMS="
youtube-dl
"
```

The removal of these programs from the script is not an acceptable change. If you'd like the functionality of not installing them please implement a command line switch and submit those changes. Otherwise these programs will remain in the script be default